@navikt/core/react/src/expansion-card/ExpansionCard.tsx

```diff
@@ -113,7 +113,7 @@ export const ExpansionCard = forwardRef<HTMLDivElement, ExpansionCardProps>(
             `navds-expansioncard--${size}`,
             {
               "navds-expansioncard--open": open ?? _open,
-              "navds-expansioncard--no-fade": !shouldFade.current,
+              "navds-expansioncard--no-anim": !shouldFade.current,
```

Comment:
Har ikke noen sterke meninger om dette, men liker selv å skrive navnene ut der man kan `navds-expansioncard--no-animation`

Reply:
ja, jeg tok en kjap en der (ville dukke opp i PR anyways, så det _bare_ var i ExpansionCard du hadde lagt det til, så her var det mulighet for en mer generisk konvensjon om jeg renamet ExpansionCard). Men skal endre til animation 👍